.github/workflows/tests_jest.yml

```
@@ -0,0 +1,33 @@
+name: Testing
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  jest:
+    name: Jest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v1
+        name: Yarn Cache
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Yarn Install
+        run: yarn --no-audit --prefer-offline
+      - name: Jest
+        run: cd tests && yarn run tests:jest-coverage
```

```suggestion
        run: yarn run tests:jest-coverage
```

No longer in test directory

---

Thanks. Updated